### PR TITLE
Clean up EOB chapter/section separated

### PR DESCRIPTION
**Pull request by helenemccarron**

### CODE REVIEW

File: books/rulesets/common/_compose.scss [hunk 1]
```diff
@@ -217,35 +217,27 @@
...
-    $map: map-get($map, $key);
-  }
-  @return $map;
-}
 
 @mixin compose_createEOBSolutions($compositePages, $solutionPage, $sectionHeaderNode) {
   $solutionSource: map-get($solutionPage, source);
   $isSeparated:  map-get($solutionPage, isSeparated);
   $sectionSeparated: map-get($solutionPage, sectionSeparated);
-  $chapterSeparated: map-deep-get($solutionPage,chapterSeparated);
```
Comment (philschatz) [inline books/rulesets/common/_compose.scss]: 💭  Why was `map-deep-get` used here previously?

File: books/rulesets/common/_compose.scss [hunk 2]
```diff
@@ -217,35 +217,27 @@
...
-@function map-deep-get($map, $keys...) {
-  @each $key in $keys {
-    $map: map-get($map, $key);
-  }
-  @return $map;
-}
 
 @mixin compose_createEOBSolutions($compositePages, $solutionPage, $sectionHeaderNode) {
   $solutionSource: map-get($solutionPage, source);
   $isSeparated:  map-get($solutionPage, isSeparated);
```
Comment (philschatz) [inline books/rulesets/common/_compose.scss]: Doh, I forgot to check if isSeparated is still used... if not, could it be removed here and in the *-config.scss files?

File: books/rulesets/common/_compose.scss (hunk 1 above)
Comment (helenemccarron) [inline books/rulesets/common/_compose.scss]: That was left-over from my attempt to use nested maps to achieve something similar to the `enum` you mentioned.

File: books/rulesets/common/_compose.scss (hunk 2 above)
Comment (helenemccarron) [inline books/rulesets/common/_compose.scss]: Ah, that one slipped, removing.

**Pull request merged**